local/src/main/java/kr/ninety/local/di/PreferencesModule.kt

```diff
@@ -0,0 +1,33 @@
+package kr.ninety.local.di
+
+import android.content.Context
+import android.content.SharedPreferences
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import kr.ninety.local.preferences.UserPreferences
+import kr.ninety.local.preferences.UserPreferencesImpl
+import javax.inject.Named
+import javax.inject.Singleton
+
+@InstallIn(SingletonComponent::class)
+@Module
+object PreferencesModule {
+
+    @Provides
+    @Singleton
+    fun provideUserPreferences(
+        @Named("userSharedPreferences") preferences: SharedPreferences
+    ): UserPreferences = UserPreferencesImpl(preferences)
+
+    @Provides
+    @Singleton
+    @Named("userSharedPreferences")
+    fun provideSharedPreferences(
```

Comment:
```suggestion
    @Provides
    @Singleton
    fun provideSharedPreferences(
```

어노테이션 제거해도 괜찮을 것 같습니다!

Reply:
요건 관련된 리뷰로 대체할게요!